recipes-kernel/linux/kernel-devsrc.bbappend

```diff
@@ -0,0 +1,216 @@
+do_install() {
```

Comment:
cp command may cause of ``` host-user-contaminated ``` QA issue[1], so using ''' cp -r --no-preserve=ownership ''' or install command is preferred. 

1: https://github.com/meta-debian/meta-debian/pull/280